polkadot-protocol/introduction/polkadot-networks.md

```diff
@@ -0,0 +1,76 @@
+
+### Development Flow
+
+Application developers and end users of Polkadot ecosystem can test many features without risking or spending their tokens on production 
+networks by using tools like [Chopsticks](#chopsticks) and TestNets. 
+
+A typical journey through the Polkadot development process might look like this:
+
+1. **Local Development Node** - For core Polkadot developers, the process begins in a local environment where you can work on upgrades and experiment with new features using a local development node. This environment allows for rapid iteration and testing in an isolated setup without the need for interacting with external networks
+
+2. **Westend** - Once the upgrades are ready, they are deployed to [Westend](#westend), Polkadot's primary TestNet. Westend is designed to mimic the conditions of a live network without the risk of using real tokens. You can extensively test features here to ensure that everything works as expected before rolling them out to Kusama or Polkadot
+
+3. **Kusama** - After rigorous testing on Westend, the next step is Kusama. Kusama serves as Polkadot's experimental version and operates as a “canary network,” where features are deployed in a real-world environment with actual economic incentives. It's a high-fidelity testing ground for any potential network upgrade
+
+4. **Polkadot** - If an upgrade passes testing on Westend and Kusama, it is finally deployed to Polkadot, the MainNet. This workflow ensures that only thoroughly vetted changes make it to production
+
+Parachain developers use local TestNets powered by tools like [Zombienet](#zombienet) and then deploy the upgrades on their respective
+parachains to a TestNet.
+
+- **Paseo** - Parachain developers often use Paseo, a community-run TestNet that mirrors Polkadot's runtime. Paseo provides parachain and dApp developers a space to test their applications with runtime environments that are closely aligned with Polkadot's without risking their projects on live networks
+
+!!!note
+    The Rococo TestNet deprecation date is October 14, 2024. Teams should use Westend for Polkadot protocol and feature testing and Paseo for chain development-related testing.
+
+## Kusama Network
+
+Kusama is the experimental development network for teams who want to innovate, move fast, and test their applications in a production-grade environment with economics and game theory involved. It comprises a relay chain with its own governance, system chains, and parachains.
+
+Native token - KSM
+
+## Test Networks
+
+The tokens for the test networks listed below are available through the [Polkadot faucet](https://faucet.polkadot.io/){target=\_blank}.
+
+### Westend
+
+Westend is a perma-TestNet that is not reset back to the genesis block. It is intended for Polkadot core developers and maintained by Parity Technologies. Westend is primarily used to test features that will be deployed onto Kusama and then to Polkadot. 
+
+Native token - WND
```

Comment:
Here, maybe we can do something like "The native token is WND. You can find out more information about WND and how to obtain some for testing purposes on the [Native Assets]() page." - and link it to the westend section: https://wiki.polkadot.network/docs/learn-DOT#getting-tokens-on-the-westend-testnet which will eventually get migrated to `Polkadot Protocol > Protocol Components > Native Assets` (as of right now)